### ISSUE TITLE
Toast notifications

### ISSUE THREAD

**Issue by nukeop** [nukeop/nuclear#169]
I'm thinking of adding toast notifications (think something like desktop notifications, but inside the player, likely in the bottom right or left corner) to display encountered errors and some async actions (such as download started/completed etc.).

Would you find these notifications helpful or annoying? Do you have any ideas regarding these you'd like me to consider?

Also, I welcome all suggestions on how to implement this best, since I've only done something like this once in another project. Should we use a third party react package?

**Comment by trekiteasy** [comment]
I have no experience with this. I think it will be interesting to have this at some point.
I believe third party package is the best choice.
Browsed a couple of packages, the most popular ans well maintained seems to be https://www.npmjs.com/package/react-toastify

**Comment by Chandankkrr** [comment]
Yes, some kind of notification to let the user know of some errors or actions in progress will be helpful. 
`react-toastify` looks pretty good but happy to explore other options as well.

**Comment by Pythonic-Rainbow** [comment]
Add a "Debug Mode" checkbox and only enable the toast when it is checked?

**Comment by nukeop** [comment]
No, it's going to be used for all kinds of notifications, not just debug ones.

**Comment by nukeop** [comment]
Ok, I started working on this on branch `feature/toasts`.

**Comment by nukeop** [comment]
![2019-02-04-115810_975x306_scrot](https://user-images.githubusercontent.com/12746779/52204375-43c7dd80-2874-11e9-9f58-4ab0f6943300.png)

Quick preview. Already released as nuclear-ui@0.0.6. There are still some minor quirks to be ironed out, but the biggest part of the task is complete.

**Comment by Chandankkrr** [comment]
Nice looks pretty good.
